pychemy/peptide_sets.py

```diff
@@ -106,3 +106,178 @@ def balanced_sets(proteins, max_set_size = 10, unique = 1):
      sets.append([p])
   return sets
 
+##################################################################################
+# Calculate ionizability of peptides based on the method used by STEPP from PNNL #
+##################################################################################
+
+from pychemy.peptides import mass_from_sequence
+import numpy as np
+import csv
+
+NUM_FEATURES = 35
+
+AA_order = ['A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L', 'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'Y']
+
+with open("resources/STEPP/STEPP_NormFactor_mean.txt") as tsv:
```

Comment:
should bundle this up into some kind of __init__ method. Maybe makes sense to put this into a class instead of just having it run once on every module instantiation